ci/ci-test.sh

```diff
@@ -44,8 +44,43 @@ dumpControllerLogs() {
   printf "\n\n"
 }
 
-# wait for zfs driver to be UP
-sleep 20
+
+isPodReady(){
+  [ "$(kubectl get po "$1" -o 'jsonpath={.status.conditions[?(@.type=="Ready")].status}' -n kube-system)" = 'True' ]
+}
+
+
+isDriverReady(){
+  for pod in $zfsDriver;do
+  isPodReady $pod || return 1
+  done
+}
+
+
+waitForZFSDriver() {
+  period=120
+  interval=1
+  
+  i=0
+  while [ "$i" -le "$period" ]; do
+    zfsDriver="$(kubectl get pods -o 'jsonpath={.items[*].metadata.name}' -n kube-system)"
```

Comment:
lets just get the pod releated to zfs-driver only. can we use -l role=openebs-zfs option? 
`kubectl get pods -l role=openebs-zfs -o 'jsonpath={.items[*].metadata.name}' -n kube-system`